binr/r2r/run.c

```diff
@@ -274,8 +281,16 @@ R_API R2RSubprocess *r2r_subprocess_start(
 	return NULL;
 }
 
-R_API void r2r_subprocess_wait(R2RSubprocess *proc) {
-	int r;
+R_API bool r2r_subprocess_wait(R2RSubprocess *proc, ut64 timeout_ms) {
+	struct timespec timeout_abs;
+	if (timeout_ms != UT64_MAX) {
+		clock_gettime(CLOCK_MONOTONIC, &timeout_abs);
+		timeout_abs.tv_nsec += timeout_ms * 1000000;
```

Comment:
maybe use a constant define instead of 1000000

Reply:
yes please

Reply:
even better, some define like `MS_TO_NSEC` and `NSEC_TO_SEC`